c/tskit/core.h

```diff
@@ -67,6 +69,33 @@ extern "C" {
 #define TSK_DBL_DECIMAL_DIG (DBL_DIG + 3)
 #endif
 
+/* We define a specific NAN value for default mutation time which indicates
+ * the time is unknown. We use a specific value so that if mutation time is set to
+ * a NAN from a computation we can reject it.
+ */
```

Comment:
We should note here where we got the bit pattern from - people will wonder where we plucked this specific value from otherwise.

Reply:
Fixed https://github.com/tskit-dev/tskit/pull/672/commits/f42d90850f749822050bc896ed2a76cd28b911c4